main.py

```diff
@@ -2,6 +2,29 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
```

Comment:
There is a typo in the actor's first name 'Kianu' and last name 'Reaves'. They should be 'Keanu' and 'Reeves' respectively to match the intended correction later in the code.